libcalico-go/lib/backend/k8s/resources/kubecontrollersconfig.go

```diff
@@ -17,11 +17,13 @@ package resources
 import (
 	"reflect"
 
-	apiv3 "github.com/projectcalico/api/pkg/apis/projectcalico/v3"
-
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
+
+	apiv3 "github.com/projectcalico/api/pkg/apis/projectcalico/v3"
+
+	v3 "github.com/projectcalico/calico/libcalico-go/lib/validator/v3"
```

Comment:
I'd recommend calling this import "validator" - "v3" is too vague and normally refers to the api/projectcalico/v3 package in other locations, so this might be confusing.